Reply/app/src/main/res/values/attrs.xml

```diff
@@ -31,4 +31,5 @@
     <!--Custom view themes/styles-->
     <attr name="bottomNavigationDrawerStyle" format="reference" />
     <attr name="bottomNavigationDrawerNavigationViewTheme" format="reference" />
+    <attr name="textAppearanceBodyExtraLarge" format="reference" />
```

Comment:
Can you add a return above this and a comment like `<!--Custom typescale attributes-->` to create a new "section"